### PR TITLE
Avro error handling

### PR DESCRIPTION
**Pull request by jmathewt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "url": "https://github.com/NYPL-discovery/discovery-bib-poster.git"
   },
   "dependencies": {
-    "avsc": "^4.1.11",
+    "avsc": "https://github.com/NYPL-discovery/avsc.git#master",
```
Comment (EdwinGuzman) [inline package.json]: You're not going to depend on NPM or on the local copy with local changes?

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Just run it through your usual linter and it should be good to go.

File: package.json (hunk 1 above)
Comment (jmathewt) [reply package.json]: Yeah we forked the avsc repo to NYPL-discovery and made changes there.

Comment (jmathewt) [reply package.json]: Cool, I will run using standard.js

**Comment by EdwinGuzman** [comment]
Thanks for running it through Standard 👍

**Review by EdwinGuzman — APPROVED** [review]

**Comment by jmathewt** [comment]
@EdwinGuzman @nonword this is great news that we don't need a forked repo and the issue was we were not calling the callback() explicitly and passing the error. If you guys can take a look at the PR and let me know that will be great.

**Pull request merged**